proto/spire/api/server/bundle/v1/bundle.proto

```diff
@@ -63,6 +63,9 @@ service Bundle {
 message GetBundleRequest {
     // An output mask indicating which bundle fields are set in the response.
     spire.types.BundleMask output_mask = 1;
+
+    // Bool value to determinate weather to use cached results or not
+    bool disable_cache = 2;
```

Comment:
I think we were going to back this out, right?

Reply:
Right! I forgot to remove it from the previous approach.